components/schematics/tsconfig.json

```diff
@@ -14,6 +14,7 @@
     "declaration": true,
     "target": "es6",
     "types": ["jest", "node"],
+    "resolveJsonModule": true,
     "baseUrl": ".",
     "paths": {
       "@angular/cdk/schematics": ["node_modules/@angular/cdk/schematics"]
```

Comment:
To what?

Reply:
will be tackled in @rowa-audil PR for the ng-add schematics in case that we have to adapt the tsconfig.json to the library structure.

for your curiosity what I meant was `'../../node_modules'`, in case that this path is relative to the file ;)